### PR TITLE
fix: query templates modification

### LINKED ISSUE #1022: Add templates for ALTER statements
for all kinds of entities: row-based table, column-based, topics.

### PR DESCRIPTION
**Pull request by antonkovalenko**
Closes #1022

- alter table in new sql section was extended
- drop table added to table
- cdc stream renamed to changefeed (like in docs)
- add changefeed option added to table context actions

### Test Status: <span style="color: green;">✅ PASSED</span>
📊 [Full Report](https://ydb-platform.github.io/ydb-embedded-ui/1579/)

| Total | Passed | Failed | Flaky | Skipped |
|:-----:|:------:|:------:|:-----:|:-------:|
| 134 | 134 | 0 | 0 | 0 |

### Bundle Size: ✅
Current: 79.21 MB | Main: 79.21 MB
Diff: +0.99 KB (0.00%)

✅ Bundle size unchanged.

<details>
<summary>ℹ️ CI Information</summary>

- Test recordings for failed tests are available in the full report.
- Bundle size is measured for the entire 'dist' directory.
- 📊 indicates links to detailed reports.
- 🔺 indicates increase, 🔽 decrease, and ✅ no change in bundle size.
</details>

### CODE REVIEW

**Comment by artemmufazalov** [comment]
In the Select name "Change data capture" is used for "changeefeed". Is it OK to have two different names in the same menu?

Screenshot 2024-11-05 at 14 10 34

File: src/containers/Tenant/utils/schemaQueryTemplates.ts [hunk 1]
```diff
@@ -68,7 +68,12 @@ WITH (
 );`;
 };
 export const alterTableTemplate = (params?: SchemaQueryParams) => {
-    return `ALTER TABLE \`${params?.relativePath || '$path'}\`
+    return `-- docs: https://ydb.tech/docs/en/yql/reference/syntax/alter_table/
+
+ALTER TABLE \`${params?.relativePath || '$path'}\`
+    -- RENAME TO new_table_name
+    -- DROP COLUMN some_existing_column
+    -- ADD COLUMN text_column Utf8 NOT NULL DEFAULT 'default_value'
```
Comment (artemmufazalov) [inline src/containers/Tenant/utils/schemaQueryTemplates.ts:76]: There is an error with such query

Screenshot 2024-11-05 at 14 18 00

**Review by artemmufazalov — CHANGES_REQUESTED** [review]

File: src/containers/Tenant/utils/schemaQueryTemplates.ts (hunk 1 above)
Comment (antonkovalenko) [reply src/containers/Tenant/utils/schemaQueryTemplates.ts]: it will be supported by default on 24-4
currently disabled
I have fixed the template

**Comment by antonkovalenko** [comment]
> In the Select name "Change data capture" is used for "changeefeed". Is it OK to have two different names in the same menu?
> 
> Screenshot 2024-11-05 at 14 10 34

yes
it's changefeed in docs
and change data capture is more popular that changefeed (based on my estimation of search results)

**Review by artemmufazalov — APPROVED** [review]

**Pull request merged**